### ISSUE TITLE
Error on Tradeoffer

### ISSUE THREAD

**Issue by nexativez** [TheAnthonyNL/SteamBot#51]
Hello is this Error normal if the 7 days timer is not finish with the authenticator ?
![image](https://cloud.githubusercontent.com/assets/7811067/13613632/483eb4f8-e56d-11e5-8af1-2870c4875584.png)

**Comment by atannerd** [comment]
yep!  Once you have authenticated for 7 days, you should get past this error.
